.github/workflows/apidiff.yml

```diff
@@ -6,11 +6,29 @@ on:
   pull_request:
 
```

Comment:
WDYT about we update the doc: https://github.com/kubernetes-sigs/kubebuilder/blob/master/CONTRIBUTING.md#where-the-ci-tests-are-configured bu adding a new section such as "Skipped jobs" and explaining what are the jobs which are or not skipped when we identify that the only docs changed?

Reply:
@NikhilSharmaWe could we not:
1 - have a job just to check the skip with this code added here
2- make the others GitHub action only check its results

Why?
Then we have this logic centralized and not duplicated across all GitHub actions.

Reply:
We discussed that it could be an improvement in a follow up as well.